src/www/ui/template/user_add.html.twig

```diff
@@ -45,12 +46,27 @@
         {{ folderListOption }}
       </select></td>
     </tr>
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <title>Toggle Password Visibility</title>
```

Comment:
The `<meta>` and `<title>` should be part of `<head>` and not of `<body>`. Please remove from here.